tests/storage/fsp/conftest.py

```diff
@@ -1,24 +1,55 @@
 import pytest
 
+FSP_REPO_FILE = '/etc/yum.repos.d/xcp-ng-runx.repo'
+
+FSP_REPO_TEXT = """\
+[xcp-ng-runx]
+name=XCP-ng runx Repository
+baseurl=http://mirrors.xcp-ng.org/8/8.2/runx/x86_64/ http://updates.xcp-ng.org/8/8.2/runx/x86_64/
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng
+"""
+
+FSP_PACKAGES = ['xcp-ng-xapi-storage']
+
 DIRECTORIES_PATH = 'directories'
 
+def install_fsp(host):
+    assert not host.file_exists('/usr/bin/runx'), \
+        "runx should not be installed on the host before all tests"
+    host.create_file(FSP_REPO_FILE, FSP_REPO_TEXT)
+    host.yum_save_state()
+    host.yum_install(FSP_PACKAGES, enablerepo='epel')
```

Comment:
1. Enabling epel is probably not necessary if we don't install runx.
2. this install will work, provided the release of the xcp-ng-xapi-storage in the runx repo is always higher than that in xcp-ng-base and xcp-ng-updates repos. If you want to ensure this works (provided we don't install xcp-ng-xapi-storage on hosts by default someday), you should disable the other repos before installing.